ansible/roles/screenly/files/screenly_utils.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+recover(){
+	sudo service screenly-viewer stop
+	rm -rf /home/pi/.screenly /home/pi/screenly_assets
+	tar -xvf "$1" -C /home/pi/
```

Comment:
Hm. The python part ensures that the file exists and the name is correct. I do not see the point of doing repeated checks. Am I mistaken?